functions/helpers.bats

```diff
@@ -2,6 +2,24 @@
 
 load helpers
 
+testthis() {
```

Comment:
This test is not static in the sense that it need external dependencies.
I think we can do better, lets think about it.

[ ] We need a test which works on a local system 
[ ] Make sure Bash get exit code are forwarded somehow

Reply:
Ok, here's a better version. Duh, this cost me nerves so please accept it like that.